website/cue/reference/components/sinks/base/honeycomb.cue

```diff
@@ -236,4 +236,21 @@ base: components: sinks: honeycomb: configuration: {
 			}
 		}
 	}
+	sample_rate_source: {
+		description: "The source of the sample rate for each event."
+		required:    false
+		type: {
+			object: options: Field: {
+				required: true
+				type: string: {}
+			}
+			string: {
+				default: "SampleTransformField"
+				enum: {
+					Disabled:             "Do not include a sample rate."
+					SampleTransformField: "Use the sample rate that is added to the event by the `Sample` transform."
```

Comment:
CI failed due to this cue file being out of date.
Once any changes are made to the config structure, you can generate this file automatically with

`make generate-component-docs` , which will create the cue file changes , and they can then be committed. This is needed on any incremental changes to the config struct member's doc comments.